src/Header/Header.js

```diff
@@ -1,13 +1,32 @@
 import React from 'react';
+import { Navbar, Nav, Button, Offcanvas, Container, NavDropdown, Form } from 'react-bootstrap';
+import logo from '../Images/company_logo.png'
+import './Header.css'
 
 export const Header = () => {
     return (
-        <header>
-            <h1>
-                Company Logo
-            </h1>
-            <button>About Us</button>
-            <button>Schedule Tour</button>
-        </header>
+        <Navbar bg='light' expand='sm'>
+            <div>
+                <Navbar.Toggle className='mx-2' aria-controls='navbarNav' />
+                <Navbar.Brand className='mx-2' href='#'>
+                    <img className='logo' src={logo} alt="" />
+                </Navbar.Brand>
+            </div>
+            
+            <Navbar.Offcanvas
+                id={`offcanvasNavbar-expand-sm`}
+                aria-labelledby={`offcanvasNavbarLabel-expand-sm`}
```

Comment:
the backtick (`) is for string interpolation in JS. Since you're not using any variables here, you can get rid of the braces and just use double quotes